boards/nxp/fmuk66-v3/src/board_config.h

```diff
@@ -343,7 +343,7 @@ __END_DECLS
 #define LED_TIM3_CH4OUT   /* PTC8  RGB_B */ PIN_FTM3_CH4_1
 
 /* This board provides a DMA pool and APIs */
-#define BOARD_DMA_ALLOC_POOL_SIZE 5120
+#define BOARD_DMA_ALLOC_POOL_SIZE 2048
```

Comment:
I believe this was sized for some worst case filesystem usage. Maybe merge only the CAN changes first and we can verify this value separately?

Reply:
@dagar - it was a WAG originally 8192 lowered to another WAG of 5120. Typical We use 3  sectors (1536) The +1 in this case, is for an additional FAT writer. Any other + (is for DMA for sensors usage) 

Can you mine the log data for it?

If not  We can keep an eye in it.  The error will be out of memory on a file operation.